libexec/pyenv-help

```diff
@@ -151,7 +151,7 @@ if [ -z "$1" ] || [ "$1" == "pyenv" ]; then
   [ -z "$usage" ] || exit
   echo
   echo "Some useful pyenv commands are:"
-  print_summaries commands $(exec pyenv-commands | sort -u)
+  print_summaries commands $(exec pyenv-commands | sort -u | grep -vFx commands)
```

Comment:
The stray `commands` is still there, just remove that?

```suggestion
  print_summaries $(exec pyenv-commands | sort -u)
```

Reply:
As I understand, the intent is to put the summary for `commands` at the top.